sklearn/preprocessing/data.py

```diff
@@ -1767,23 +1770,25 @@ class OneHotEncoder(BaseEstimator, TransformerMixin):
         Will return sparse matrix if set True else will return an array.
 
     handle_unknown : str, 'error' or 'ignore'
-        Whether to raise an error or ignore if a unknown categorical feature is
-        present during transform.
+
+        - 'ignore': Ignore all unknown feature values.
+        - 'error': Raise an error when the value of a feature is unseen during
+          `fit` and out of range of values seen during `fit`.
+        - 'error-strict': Raise an error when the value of a feature is unseen
+          during`fit`.
+
+    copy : bool, default=True
+        If unset, `X` maybe modified in space.
 
     Attributes
     ----------
-    active_features_ : array
-        Indices for active features, meaning values that actually occur
-        in the training set. Only available when n_values is ``'auto'``.
+    feature_index_range_ : array, shape [n_feature, 2]
+        `feature_index_range_[i]` specifies the range of column indices
+        occupied by the feature `i` in the one-hot encoded array.
 
-    feature_indices_ : array of shape (n_features,)
-        Indices to feature ranges.
-        Feature ``i`` in the original data is mapped to features
-        from ``feature_indices_[i]`` to ``feature_indices_[i+1]``
-        (and then potentially masked by `active_features_` afterwards)
-
-    n_values_ : array of shape (n_features,)
-        Maximum number of values per feature.
+    one_hot_feature_index_ : array, shape [n_features_new]
+        `one_hot_feature_index_[i]` specifies which feature of the input
```

Comment:
This is somewhat redundant to `feature_indices_` ( as it ignores non-categorical variables) and completely redundant with the current `feature_index_range_`. I realize now that this is a mistake. From your comment on #6602, you preferred something like `one_hot_feature_index_ ` and wanted to deprecate `feature_indices_`,correct ? Should I completely remove `feature_index_range_` then ?